include/kernel.h

```diff
@@ -1951,23 +1951,8 @@ static inline s64_t k_uptime_delta(s64_t *reftime)
 	return delta;
 }
 
-/**
```

Comment:
Do not remove the documentation please, leave in place and add doxygen @deprecated command

Reply:
I recall being told the opposite at one point, but I agree with this approach.  Done (though I replaced the blatently false paragraph in the old documentation with the truth).